File: JAVA/1672. Richest Customer Wealth.java
class Solution {
    public int maximumWealth(int[][] a) {
        int i,j,s,m=0;
        for(i=0;i<a.length;i++)
        {
            s=0;
            for(j=0;j<a[0].length;j++)
            {
                s+=a[i][j];
            }
            if(m<s)
            m=s;
        }
        return m;
    }
}

File: C++/1672. Richest Customer Wealth.cpp
class Solution {
public:
    int maximumWealth(vector<vector<int>>& a) {
        int m=0,i,j,s;
        for(i=0;i<a.size();i++)
        {
            s=0;
            for(j=0;j<a[i].size();j++)
            {
                s+=a[i][j];
            }
            if(m<s)
            m=s;
        }
        return m;
    }
};